examples/custom-ipfs-repo/index.js

```diff
@@ -0,0 +1,77 @@
+'use strict'
+
+const series = require('async/series')
+const IPFS = require('ipfs')
+
+// Create our custom options
+const customRepositoryOptions = {
+
+  /**
+   * Storage Backends are fully customizable. Each backend can be stored in seperate services,
+   * or in a single service. Options can be passed into the datastores via the storageBackendOptions
```

Comment:
It’s not really the *backend* that’s stored in a service, right? Maybe something like this would be a little clearer?

> IPFS nodes store different information in separate `storageBackends`, or datastores. Each storage backend can use the same type of datastore or a different one — you could store your `keys` in a levelDB database while everything else is in files, for example. (See https://github.com/ipfs/interface-datastore for more about datastores.)